pkgs/development/libraries/libiio/cmake-fix-libxml2-find-package.patch

```diff
@@ -0,0 +1,13 @@
+diff --color -ur a/CMakeLists.txt b/CMakeLists.txt
+--- a/CMakeLists.txt	2021-05-30 13:46:22.256040282 +0200
++++ b/CMakeLists.txt	2021-05-30 14:15:42.530181216 +0200
+@@ -333,7 +333,7 @@
+ # So, try first to find the CMake module provided by libxml2 package, then fallback
+ # on the CMake's FindLibXml2.cmake module (which can lack some definition, especially
+ # in static build case).
+-find_package(LibXml2 QUIET NO_MODULE)
++find_package(LibXml2 QUIET MODULE)
```

Comment:
Would that not be an issue with libxml2 itself?

I do see the config present at /nix/store/7xd3qxnfqd9gy2wxd5i4q2v4r9201imm-libxml2-2.9.12-dev/lib/cmake/libxml2/libxml2-config.cmake

Reply:
Seems like a problem with multiple-outputs of libxml2. This one configures fine for me:

```
with import <nixpkgs> {};
  
libiio.override {
  libxml2 = libxml2.overrideAttrs (old: {
    postFixup = old.postFixup + ''
      substituteInPlace "$dev"/lib/cmake/libxml2/libxml2-config.cmake \
        --replace "set(LIBXML2_LIBRARY_DIR " "set(LIBXML2_LIBRARY_DIR \"$out/lib\")#"
    '';
  });
}
```